templates/default/mount-hopsfs.sh.erb

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+set -e
+
+mount_point=<%= node['hops']['fuse']['mount_point'] %>
+mount_bin="<%= @fuse_mount_bin %>"
+common_params="--stageDir <%= node['hops']['fuse']['staging_folder'] %>  <%= @nn_address %>:<%= node['hops']['nn']['port'] %>  <%= node['hops']['fuse']['mount_point'] %>  > <%=node['hops']['logs_dir']%>/fuse-mount.log 2>&1"
+
+<% if node['hops']['tls']['enabled'].eql? "true" -%>
```

Comment:
Use `.casecmp?()` which does a case insensitive comparison of the strings

Reply:
fxd